test/path.js

```diff
@@ -28,6 +28,19 @@ describe('path', function() {
     eq(R.path(['j', 1], obj), undefined);
   });
 
+  it('takes a path that contains indeces into arrays that might be negative', function() {
+    var obj = {
+      a: [[{}], [{x: 'first'}, {x: 'second'}, {x: 'third'}, {x: 'last'}]]
+    };
+    eq(R.path(['a', 0, 0], obj), {});
+    eq(R.path(['a', 0, 1], obj), undefined);
+    eq(R.path(['a', 1, 0, 'x'], obj), 'first');
+    eq(R.path(['a', 1, 1, 'x'], obj), 'second');
+    eq(R.path(['a', 1, -2, 'x'], obj), 'third');
```

Comment:
I know that these tests are building upon what was already there, but I think slowly the signal is getting lost in the noise.  There should be other tests for deep paths, so a demonstration of negative indices should be simpler, perhaps something like this:

```js
eq(R.path(['x', -2], {x: ['a', 'b', 'c', 'd']}), 'c')
```
along with

```js
eq(R.path([-1, 'y'], [{x: 1, y: 99}, {x, 2, y: 98}, {x: 3, y: 97}]), 97)
```